### ISSUE TITLE
[Tinkers Construct] Auto-Smelt + Fortune III Issue

### ISSUE THREAD

**Issue by Lankoran** [WTFGeeks/Natural-Magic#64]
I've noticed that my TC Pick, with Auto-Smelt and Fortune III doesn't work on Forestry Copper and Tin Ores, instead of returning multiple ingots as it should, it returns apatite from Forestry.

The copper/tin added by Erebus doesn't smelt at all, but the Tinkers Construct ones work fine.

**Comment by WTFGeeks** [comment]
I'll have a look into this for you.

**Comment by BronyfromOz** [comment]
Auto-Smelt and Fortune III pick on Sunstone Ore (Ars Magica) returns vinteum dust.

**Comment by WTFGeeks** [comment]
Yeah these are both issues on tinkers end, not much I can do until it is fixed.
